### PR TITLE
Fix Parker mode

### LINKED ISSUE #257: parker mode does not work
today we wanted to test the "parker mode", which goes like this:

The shifter performs the normal startup and is available during the night for calls. In the morning, the "parker" clicks the "Ready for shutdown!" button on the SH website (https://shifthelper.app.tu-dortmund.de/) in the time window between 30min and 20min before shutdown.

This way the SH would not call the shifter, since it assumes the shifter is already awake and "ready for shutdown". The parker would then observe/supervise the shutdown and fill the checklist. Any problems during the shutdown, which normally would lead to a call to the shifter, still lead to a call to the shifter. E.g. if during shutdown main.js throws an exception due to a drive problem, then the shifter is still called.

We believe the parker mode is especially important for the short nights in summer, where startup is very late and shutdown is very early.

The test failed, because ...

... the SH code is too strict. It does not only check if anybody is "ready for shutdown" but that the shifter is ready for shutdown. This was done to avoid that somebody in the FACT collaboration clicks this button in the morning accidentally and this way a SH-call is not send out.

So for parker mode, we have to change the SH code here:

https://github.com/fact-project/shifthelper/blob/a94949748acd2aa5221d573a568d2346437f4254/shifthelper/conditions.py#L275-L278

to simply:
```python
   return not awake
```

This way anybody in the collaboration can press the "Ready for Shutdown!" button in the time window between 30min and 20min before shutdown and thus take the role of the parker.

To assert everything still works after this minor change, the SH will still undergo the manual test suite here:

https://github.com/fact-project/shifthelper_smartfact_mockup_test

The modification in the SH and the following manual test will be conducted today around 9 UTC.

So tomorrow we can test the parker mode again.

### PR DESCRIPTION
**Pull request by dneise**
This should fix #257.

It was branched off `new_fsc` version, which is open PR #252. 
I've set the base to be `new_fsc`, in order to show how small the diff really is.

### CODE REVIEW

**Comment by dneise** [comment]
Max is in Holiday so I'd like to have Kai or Sebastian to review this.

**Comment by dneise** [comment]
In addition to what was needed to make the parker mode work, I renamed some variables from "awake" to "ready for shutdown". Since we renamed the button on the website, and this might create some continuity here. 

However it is not complete. The url of the SH-webinterface to request who is "ready for shutdown" is still called "/iAmAwake". And the SH function to make a get request on this url is still called `fetch_users_awake()`.

Completely getting rid of the "awake" name and move to "ready_for_shutdown", involves modifications in both repositories and in the config file, which is still another repository, and on the server in dortmund. 

This is more than what I dare to do quickly.

File: shifthelper/conditions.py [hunk 1]
```diff
@@ -274,17 +274,14 @@ def is_no_shift_at_the_moment():
 
 
 @log_call_and_result
-def is_nobody_awake():
-    '''Parker not Awake'''
-    awake = {}
+def is_nobody_ready_for_shutdown():
```
Comment (mackaiver) [inline shifthelper/conditions.py]: The method name and the return value `not ready_ready_for_shutdown` suggest a double negative. To my puny human mind `is_anybody_ready_for_shutdown` can be converted more quickly into the corresponding logical formula. But thats just nitpicky stuff. Not important.

**Comment by mackaiver** [comment]
> Completely getting rid of the "awake" name and move to "ready_for_shutdown", involves modifications in both repositories and in the config file, which is still another repository, and on the server in dortmund.

Sounds kind of complicated. But I guess there is no way around that.

File: shifthelper/conditions.py (hunk 1 above)
Comment (dneise) [reply shifthelper/conditions.py]: I agree, this is complicated.

There is however an "interface" defined here. All these conditions can be listed in a "list of conditions", e.g. like here: 
https://github.com/fact-project/shifthelper/blob/a94949748acd2aa5221d573a568d2346437f4254/shifthelper/__main__.py#L216-L223

If all conditions evaluate to True, then we emit a call. So we are not really free here, if we want to stick to these condition lists. Maybe this is a hint that these condition lists were not a good idea. What do you think? 

However, removing these condition lists for something else is a larger modification ... so, we should have a different PR for that.

Comment (mackaiver) [reply shifthelper/conditions.py]: > If all conditions evaluate to True, then we emit a call. So we are not really free here, if we want to stick to these condition lists. Maybe this is a hint that these condition lists were not a good idea. What do you think?

Ah cool. Was not aware of that. Having these lists seems very reasonable to me. 
In theory one could put a `not` in front of these conditions in the list right? Might make it a bit more readable for me personally.

Also, if one concatenates all those lists and ends up in  [NNF](https://de.wikipedia.org/wiki/Negationsnormalform) or [DNF](https://de.wikipedia.org/wiki/Disjunktive_Normalform),  that would certainly be interesting from an academic standpoint (theory of control systems and all that), but probably not of much use for anything else.  

In any case, I propose to leave it like that for now. I Seems very reasonable to me.

Comment (dneise) [reply shifthelper/conditions.py]: perfect ... thanks for this very interesting feedback.

**Pull request merged**